zmon_aws_agent/aws.py

```diff
@@ -783,3 +783,44 @@ def get_limits(region, acc, apps, elbs):
     entity.update(limits)
 
     return entity
+
+
+def get_sqs_queues(region, acc):
+    sqs_queues = []
+
+    try:
+        sqs_client = boto3.client('sqs', region_name=region)
+        list_queues_response = call_and_retry(lambda: sqs_client.list_queues())
+        for queue_url in list_queues_response['QueueUrls']:
```

Comment:
If SQS queues info doesn't change that often, and I guess the main target is to write checks based on the ARN, then we can make a call every 10 or 20 mins, just to somehow reduce API rate limit errors.